### PR TITLE
Workflow Fix for JDBC & Workbench

### PR DESCRIPTION
**Pull request by davidcui-amzn**
*Issue #, if available:*
N/A
*Description of changes:*
Fix workflow credentials for JDBC and s3 bucket name for workbench

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/sql-jdbc-release-workflow.yml [hunk 1]
```diff
@@ -37,6 +37,13 @@ jobs:
         run: |
           ./gradlew publishShadowPublicationToInternal-releasesRepository '-Dorg.gradle.jvmargs=--add-modules java.xml.bind' -Dbuild.snapshot=false
 
+      - name: Configure Staging AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
```
Comment (joshuali925) [inline .github/workflows/sql-jdbc-release-workflow.yml:41]: jdbc needs `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to build?

**Review by joshuali925 — APPROVED** [review]

File: .github/workflows/sql-jdbc-release-workflow.yml (hunk 1 above)
Comment (davidcui-amzn) [reply .github/workflows/sql-jdbc-release-workflow.yml:41]: Currently the build is failing with an `Access denied` error, after a short working session with Sreekar from infra team he suggested using this workaround

**Review by dai-chen — APPROVED** [review]

**Comment by joshuali925** [comment]
got it, could you also add back `cd kibana/plugins/workbench` in here above line 62? I think it was accidentally removed in staging bucket change, without `cd` the working directory is not right and `ls` doesn't work https://github.com/opendistro-for-elasticsearch/sql/blob/2f2d8c31f08f9a0a3073c4754a6755264366dec3/.github/workflows/sql-workbench-release-workflow.yml#L61-L62

**Comment by davidcui-amzn** [comment]
> got it, could you also add back `cd kibana/plugins/workbench` in here above line 62? I think it was accidentally removed in staging bucket change, without `cd` the working directory is not right and `ls` doesn't work
> 
> https://github.com/opendistro-for-elasticsearch/sql/blob/2f2d8c31f08f9a0a3073c4754a6755264366dec3/.github/workflows/sql-workbench-release-workflow.yml#L61-L62

Will do, maybe that is why the workbench is failing too

**Pull request merged**